lib/messenger.rb

```diff
@@ -0,0 +1,13 @@
+# Sends messages using Twilio integration
+class Messenger
+  def confirm(phone)
+    client = Twilio::REST::Client.new(ENV['TWILIO_ACCOUNT_SID'], ENV['TWILIO_AUTH_TOKEN'])
```

Comment:
The twilio client could be a dependency on the `Messenger` class if we wanted to test this object in isolation too. (The test for that could as simple as checking the create method gets called)